benefits/eligibility/templates/eligibility/start.html

```diff
@@ -6,37 +6,45 @@
 
 {% block main_content %}
   <div class="container">
```

Comment:
I realized this code here will be soon deleted with the new base templates, etc. but that's okay -- I needed to see what the Media List component looks like within the proper column-widths and rows. And I wanted to be able to test the `<h2 class="media-title p-sm">` too. Having this code here will still allow us to test what this page looks like on dev and therefore our phones, etc.